Modules/SubModule_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -19,17 +20,20 @@ function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux)
 %
 % EXAMPLE: xASL_adm_GzipAllFiles('/MyStudy');
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% Copyright 2015-2020 ExploreASL
+% Copyright 2015-2021 ExploreASL
 
 
     %% ----------------------------------------------------
     %% 0) Admin
-    if nargin<2 || isempty(bFolder)
+    if nargin<2 || isempty(EXTERNAL)
+        EXTERNAL = '';
```

Comment:
should default to `[]` for `~isempty()` to work I believe

Reply:
So this becomes 

`if nargin<2`
`pathExternal = [];`
`end`

Reply:
Removed 👍